### DISCUSSION TITLE
7.4.4, when?

### DISCUSSION THREAD

**Discussion by Nicelittlewood** [betalgo/openai#477] [General]
question is in the title. :)

**Comment by kayhantolga** [comment]
Just published, but the milestones have been changed :/

**Reply by Nicelittlewood** [reply]
i see, but is not what i hope :(

i'll keep waiting for the assistant' apis.

TY

**Comment by kayhantolga** [comment]
Sorry for the delay. I have been working on a tool, which is why I couldn't find enough time to implement beta features. However, I promise that the tool will be worth the wait.
